number_trivia_app/test/widget_test.dart

```diff
@@ -0,0 +1,30 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility in the flutter_test package. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:number_trivia_app/main.dart';
+
+void main() {
```

Comment:
Consider removing this file

Reply:
This file is removed on the following pull requests!

Reply:
Remove it anyway. The purpose is to force conflicts on future PRs (Pull Requests)